diagnostic.md

```diff
@@ -40,25 +61,29 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
   ```md
-    # < Your Response Here >
+    The serializer lists the attributes associated with a table and it defines what information to obtain from a related table
   ```
 
   ```rb
   class ProfileSerializer < ActiveModel::Serializer
+
+    def profiles
```

Comment:
`has_many :movies` should be used since we might want more than the movie `id`s.